docs/permissions/data.md

```diff
@@ -50,12 +50,25 @@ Permission levels include:
 - [Impersonated](#impersonated-view-data-permission)
 - [Blocked](#blocked-view-data-permission)
 
+View data permission settings apply to different levels in your database:
+
+| View data permission | Database | Schema | Table |
+| -------------------- | -------- | ------ | ----- |
+| Can view             | ✅        | ✅      | ✅     |
+| Granular             | ✅        | ✅      | ❌     |
```

Comment:
Do you think "Granular" is useful here? It isn't really a permission-level. It is more like a consequence

Reply:
Agree, but thought it might be worth including "Granular" because it's a selection in dropdown menus when setting perms, and the setting itself signals that the setting is defined at a more granular level. I'll star it in the table and explain.